pkg/users/group-actions.jsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2020 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from 'cockpit';
+import React, { useState } from 'react';
+
+import { Dropdown, DropdownItem, DropdownSeparator, KebabToggle } from '@patternfly/react-core';
+
+import { delete_group_dialog } from "./delete-group-dialog.js";
+import { rename_group_dialog } from "./rename-group-dialog.js";
+
+const _ = cockpit.gettext;
+
+export const GroupActions = ({ group, accounts, isMainPage }) => {
+    const [isKebabOpen, setKebabOpen] = useState(false);
+    const actions = [];
+
+    if (isMainPage) {
+        actions.push(
+            <DropdownItem key="edit-group"
+                          onClick={ev => { ev.preventDefault(); cockpit.location.go(["group", group.name]) }}>
+                {_("Edit group")}
+            </DropdownItem>
+        );
+    } else if (group.isUserCreatedGroup) {
+        actions.push(
+            <DropdownItem key="rename-group"
+                          onClick={() => { setKebabOpen(false); rename_group_dialog(group.name) }}>
+                {_("Rename")}
+            </DropdownItem>
+        );
+    }
+
+    if (group.isUserCreatedGroup) {
+        actions.push(
+            <DropdownSeparator key="separator" />,
+            <DropdownItem key="delete-group"
+                          className={group.uid === 0 ? "" : "delete-resource-red"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230222-131216-482495e4-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)